Automattic/jetpack
# Autoloader: detect a woo plugin activation request

Fixes #19624

#### Changes proposed in this Pull Request:
* Add a new method, `find_activating_plugins_in_woo_request`, which attempts to detect plugins that are activating via a Woo request. Add these activating plugins to the active plugins list.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Use this version of the autoloader with Woo.
2. Go through the Woo onboarding flow and select to install both WooCommerce Payments and Jetpack.
3. Verify that both WooCommerce Payments and Jetpack were installed and activated.
